pixi.toml

```diff
@@ -54,7 +54,8 @@ test = "pytest tests/integration --backend=postgres"
 coverage = "pytest tests/integration --cov=datajudge --cov-report=xml --cov-append --backend=postgres"
 
 [feature.db2.dependencies]
-ibm_db = "*"
+# Upper bounded due to https://github.com/ibmdb/python-ibmdb/issues/986
```

Comment:
@ivergara See this :)

Credits to SImon

Reply:
Great